doc/source/data/transforming-datasets.rst

```diff
@@ -373,6 +373,97 @@ The following output types are allowed for per-row UDFs (e.g.,
     :start-after: __writing_simple_out_row_udfs_begin__
     :end-before: __writing_simple_out_row_udfs_end__
 
+.. _datasets-groupbys:
+
```

Comment:
what about https://docs.ray.io/en/master/data/dataset-ml-preprocessing.html#random-shuffle and https://docs.ray.io/en/master/data/dataset-ml-preprocessing.html#random-block-order? Should we add here as well?

Reply:
For `randomize_block_order`, I've added a callout in "Shuffling Data" section of the AIR user guide.

For `random_shuffle`, I'll follow Clark's suggestion:
> As for the shuffling docs, why don't we add a shuffle section to the "Transforming Datasets" feature guide that only details the ds.random_shuffle() API (basically the section that we have in the current "ML Preprocessing" guide), and have it cross-link to the AIR docs on shuffling + call out that local shuffling is a good alternative, but delegate details to the AIR level docs?